beetsplug/gmusic.py

```diff
@@ -62,6 +68,14 @@ def upload(self, lib, opts, args):
         self.m.upload(filepaths=files)
         ui.print_(u'Your files were successfully added to library')
 
+    def autoupload(self, session, task):
+        items = task.imported_items()
+        files = [x.path.decode('utf-8') for x in items]
```

Comment:
Although I know it’s what the code above does too, this `decode` call is pretty worrisome and will probably break on some inevitable non-UTF-8 filenames. Can we please factor this out into a helper function so it’s possible to fix both at once?